cli/command/trust.go

```diff
@@ -12,6 +12,10 @@ var (
 	untrusted bool
 )
 
+func init() {
+	untrusted = !getDefaultTrustState()
```

Comment:
This is necessary to test any code that reads `command.IsTrusted()`. The default is trusted, which requires mocking of a bunch more client calls.